bin/run_tests.py

```diff
@@ -4,7 +4,7 @@
 import os, sys, subprocess, shutil, json
 from glob import glob
 
-from . import run_test
+import run_test
```

Comment:
I don't know if there is a better way of doing this 'import from the same folder'?

Reply:
No, there isn't. Also since these are utility scripts I'd prefer using the shell interface instead, something like `subprocess.check_call([sys.executable, 'bin/run_test.py', arg1, arg2])`

Reply:
There you go, then. I'd chosen for this such that the whole argument parsing (and extra output 'Project built successfully.') was not executed every time.

Have a look, maybe the `if __name__ == '__main__':` construction was also unnecessary, then.